### ISSUE TITLE
Version 2.0.3 appears to be a breaking change

### ISSUE THREAD

**Issue by yhnavein** [jryans/timers-browserify#26]
Hi,

We stumbled upon an error caused by version **2.0.3** of the `timers-browserify`. 
You may seen a following error throwed by `webpack` (through the `node-libs-browser`):

```txt

ERROR in ./node_modules/timers-browserify/main.js
Module not found: Error: Can't resolve 'global' in 'C:\bc\components\app-seed\node_modules\timers-browserify'
 @ ./node_modules/timers-browserify/main.js 52:13-30
 @ ./node_modules/@britishcouncil/common/dist/bc-sharedcomponents.js
 @ ./src/app/index.route.ts
 @ ./src/app/index.module.ts
 @ ./src/index.ts
 @ multi (webpack)-dev-server/client?http://localhost:3000 webpack/hot/dev-server ./src/index.ts
webpack: Failed to compile.

```

Please mark release `2.0.3` as a breaking release.

Cheers,

**Comment by jryans** [comment]
Hmm, I don't think I am following entirely... The new dependency is declared in `package.json`. Why are you seeing an error?

**Comment by arutkowski00** [comment]
That's a good question, why. I have the same issue, I had to downgrade `timers-browserify` to 2.0.2

**Comment by jryans** [comment]
Hmm, strange. Well, since Webpack is a more important use case than the one 2.0.3 was meant to help with, I'll revert it for now until we better understand the issue here.

**Comment by jryans** [comment]
Okay, I have released 2.0.4 which reverts the problematic change.
